recipes/leftbar/general_row/general_row.vue

```diff
@@ -343,7 +343,9 @@ export default {
     },
 
     getAriaLabel () {
-      return this.ariaLabel ? this.ariaLabel : `${this.description} ${this.unreadCountTooltip} ${this.dndTextTooltip}`;
+      return this.ariaLabel
+        ? this.ariaLabel
+        : [this.description, this.unreadCountTooltip, this.dndTextTooltip].filter(str => !!str).join(' ');
```

Comment:
With the util function this will look better it will improve the readability of this
` : [this.description, this.unreadCountTooltip, this.dndTextTooltip].filter(str => !!str).join(' ');`